server/test/user/user.e2e-spec.ts

```diff
@@ -0,0 +1,297 @@
+import {HttpModule, INestApplication} from '@nestjs/common';
+import {Test} from '@nestjs/testing';
+import request from 'supertest';
+import {UserModule} from '../../src/port/module/user.module';
+import {User} from '../../src/domain/user/user.entity';
+import {DatabaseModule} from '../../src/port/module/database.module';
+import {ConfigModule, ConfigService} from 'nestjs-config';
+import * as path from 'path';
+import {getConnection} from 'typeorm';
+import * as assert from 'assert';
+
+// todo: implement ExceptionHandler, apply custom Error
+
+describe('UserController', () => {
+    let app: INestApplication;
```

Comment:
plz read https://github.com/nestjs/nest/issues/1843 this to rollback transactions

Reply:
i used `dropSchema` option at config file.

it's different from the reference, but i wanted to gather all the configuration in config file which is not applied at the reference